packages/block-editor/src/hooks/style.js

```diff
@@ -82,13 +74,13 @@ export function getInlineStyles( styles = {} ) {
 		const path = STYLE_PROPERTY[ propKey ].value;
 		const subPaths = STYLE_PROPERTY[ propKey ].properties;
 		// Ignore styles on elements because they are handled on the server.
-		if ( has( styles, path ) && 'elements' !== first( path ) ) {
+		if ( has( styles, path ) && 'elements' !== path[ 0 ] ) {
```

Comment:
I inferred from the values in [__EXPERIMENTAL_STYLE_PROPERTY](https://github.com/WordPress/gutenberg/blob/5654e3dff70d1cd84d7f74aa6d211683a99ff366/packages/blocks/src/api/constants.js#L17), but it might be safer to add optional chaining here as well just in case 👍

Reply:
Ah, didn't do the lookup, I think in this case we could do without the optional - would be certain to not follow with it if we had typescript here and we're certain value is always an array. Will leave this up to you if we keep the optional chaining or not :)